app/controllers/support/users_controller.rb

```diff
@@ -1,30 +1,20 @@
 class Support::UsersController < Support::BaseController
   before_action :set_user, except: %i[new create search results]
   before_action { authorize User }
+  before_action :set_school_if_present, only: %i[new create]
+  before_action :set_responsible_body_if_present, only: %i[new create]
+  before_action :deny_access_if_school_cannot_invite_users, only: %i[new]
```

Comment:
Not specifically related to this PR and I am aware it was already handled this way, but shouldn't this be handled in a Pundit policy. Maybe one for a new card and PR.

Reply:
Well, i think it is not exactly an authorisation policy. It is not about authorising the action to the current but wether the school itself is set to invite users or not.